src/labels/CustomLabels.labels

```diff
@@ -5093,6 +5093,22 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <shortDescription>Button to process a batch.</shortDescription>
         <value>Process Batch</value>
     </labels>
+    <labels>
+        <fullName>bgeRetirementWarningBody</fullName>
+        <categories>Batch Data Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Explanation of Batch Gift Entry retirement &amp; recommendations</shortDescription>
+        <value>On July 13, 2021, legacy Batch Gift Entry will be retired and will no longer work in NPSP. If you have any in-progress batches at that time, you can still access those records, but you can't process them using Batch Gift Entry. To ensure a smooth transition, enable and use the newer Gift Entry in advance of July 13th. Learn more about Gift Entry in the &lt;a href=&apos;https://powerofus.force.com/NPSP_Documentation&apos; target=&apos;_blank&apos;&gt;NPSP Documentation&lt;/a&gt; on the Power of Us Hub.</value>
```

Comment:
Does the markup within this label cause trouble for anything that attempts to block XSS attacks here? Do we have an improved system for including markup w/in labels? (@psadie-sfdo I feel like you know about this?)

Reply:
I believe this is good! My understanding from our last check-in with the localization folks (quite some time ago) is that the danger from this approach is if we ourselves were to write JS directly in there or if were to unescape like `<apex:outputText escape="false" value="{!Label. bgeRetirementWarningBody}">` wherever this label is used.

Reply:
Thank you! Sounds good, marking this resolved!